### ISSUE TITLE
Often the app would show "App restart required..." error and would quit after user hits "OK"

### ISSUE THREAD

**Issue by vitalii-tym** [opentrace-community/opentrace-ios#9]
`CBCEntralManager` initially starts with state `uknown` and then changes to `poweredOn` in a moment. But the app happens to check the manager's state and shows an alert just before the state changes. This results in user getting this false error message (and sometimes two of them):

![Screen Shot 2020-04-15 at 23 36 58](https://user-images.githubusercontent.com/5410835/79386196-78542080-7f72-11ea-8e2b-4910c7858fd2.png)

Since the error has `exit(0)` command under the OK button, the app would be halted instantly. So that **user has no chance to get the app up and running**.

Also **the `exit(0)` command mentioned above is forbidden is having it in the app would result app being rejected by Apple**.

Notes: the above behavior is for "release" configuration, in "debug" configuration the error message is a little different and there is no `exit(0)` command. But the false error still appears.

Expected: since there are already some UI elements inside the app indicating about bluetooth problems (if any) the error can be completely removed.

**Comment by gibtang** [comment]
Switching off Bluetooth triggers this error message too and then tapping the button in the modal will close the app. App HIG seems to discourage this type of behaviour. https://gamedev.stackexchange.com/questions/17709/can-an-ios-application-have-an-explicit-quit-button

The better option is to have a button that opens the Settings > Bluetooth screen. I can submit a PR for this if you are ok with merging my PR

**Comment by hongshaoyang** [comment]
The alert shown in the screenshot above is from this: 

https://github.com/opentrace-community/opentrace-ios/blob/eaa667363a0a4421253a5e7c067f0d37131cf875/OpenTrace/Bluetrace/BluetraceManager.swift#L42-L46

**Comment by gibtang** [comment]
Yup, users should not be allowed to kill any app using programmatic means as mentioned by Apple. Better way is to switch to the BT settings to let use enable BT. And if BT is still not enabled, then popup alert nagging user to switch it on
